dataflow/src/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -74,6 +74,18 @@
     protected IdentityHashMap<Block, S> elseStores;
 
     /**
+     * Number of times every block has been analyzed since the last time widening was applied. Null,
+     * if maxCountBeforeWidening is -1 which implies widening isn't using for this analysis.
+     */
+    protected IdentityHashMap<Block, Integer> blockCount;
```

Comment:
All of the other hash maps fields in this class are re-created on every init.  Why is clearing a map faster than creating a new one?